iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -80,6 +80,22 @@ Fact.prototype.readableValue = function() {
             v = formatNumber(v,d) + " " + this.unit().valueLabel();
         }
     }
+    /* If the text value appears to start with an HTML tag, treat as HTML and
+     * use the text content as the value.  It would be better to apply this
+     * based on data type, but we don't currently have access to that in the
+     * report data.
+     */
+    else if (v.match(/^[\s\u00a0]*<([a-zA-Z_][\w.-]*:)?[a-zA-Z_][\w.-]*(\s|>|\/>)/)) {
```

Comment:
It seems like this should be based on `<ix:nonNumeric escape="true">`.  If that attribute isn't currently available in the JSON, it seems like we should make it available.

Reply:
Good point!  I'd completely forgotten that the fact that we're coming from iXBRL means that we have the escaped attribute, which actually preferable to relying on a datatype and heuristic.  It should be possible to have the JS pull this straight from the iXBRL, rather than adding it to the JSON.

Reply:
This is done in 163e12f